src/js/components/molecules/footer-navigation-category/footer-navigation-category.vue.html

```diff
@@ -0,0 +1,8 @@
+<div class="footer-navigation-category__column" v-if="items.length">
```

Comment:
`footer-navigation-category` or `footer-navigation__category`… `column` is too presentational.